From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <kleinschmidtorama@gmail.com>
Date: Fri, 6 Sep 2019 12:51:11 -0400
Subject: add temporary logging for ci issue


diff --git a/net/dns/host_resolver_proc.cc b/net/dns/host_resolver_proc.cc
index 0824540a63a37cc84a0dc0923f3b3d6e72fb477b..b39d6094d4d6c90d1632d43413f0f26b49e22227 100644
--- a/net/dns/host_resolver_proc.cc
+++ b/net/dns/host_resolver_proc.cc
@@ -15,6 +15,7 @@
 #include "net/dns/dns_reloader.h"
 #include "net/dns/dns_util.h"
 #include "net/dns/host_resolver.h"
+#include <base/debug/stack_trace.h>
 
 #if defined(OS_OPENBSD)
 #define AI_ADDRCONFIG 0
@@ -227,8 +228,10 @@ int SystemHostResolverCall(const std::string& host,
   }
 
   if (err) {
+  base::debug::StackTrace().Print();
 #if defined(OS_WIN)
     err = WSAGetLastError();
+    LOG(INFO) << "HEY WSAGetLastError was:" << err;
 #endif
 
     // Return the OS error to the caller.
@@ -238,14 +241,21 @@ int SystemHostResolverCall(const std::string& host,
     // If the call to getaddrinfo() failed because of a system error, report
     // it separately from ERR_NAME_NOT_RESOLVED.
 #if defined(OS_WIN)
-    if (err != WSAHOST_NOT_FOUND && err != WSANO_DATA)
+    LOG(INFO) << "HEY OS_WIN defined";
+    if (err != WSAHOST_NOT_FOUND && err != WSANO_DATA) {
       return ERR_NAME_RESOLUTION_FAILED;
+    }
+    LOG(INFO) << "err is WSAHOST_NOT_FOUND or WSANO_DATA";
 #elif defined(OS_POSIX) && !defined(OS_FREEBSD)
-    if (err != EAI_NONAME && err != EAI_NODATA)
-      return ERR_NAME_RESOLUTION_FAILED;
+    LOG(INFO) << "HEY OS_POSIX defined";
+    if (err != EAI_NONAME && err != EAI_NODATA) {
+       LOG(INFO) << "HEY err != EAI_NONAME && err != EAI_NODATA";
+      return err;
+    }
+    LOG(INFO) << "HEY err is EAI_NONAME or err is EAI_NODATA";
 #endif
-
-    return ERR_NAME_NOT_RESOLVED;
+     LOG(INFO) << "HEY returning: ERR_NAME_NOT_RESOLVED";
+    return err;
   }
 
 #if defined(OS_ANDROID)
